src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Color[] colors = Color.values();
+        Random random = new Random();
+        int index = random.nextInt(colors.length);
+        return String.valueOf(colors[index]);
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/12864128/e2341b2d-187e-4eb3-8807-ba11e2260ca1)
See checklist: https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery

Reply:
comment not fixed, use .name() method